.gitignore

```diff
@@ -1,5 +1,5 @@
-node_modules
-yarn.lock
+**node_modules
+**yarn.lock
```

Comment:
Does `**` really matter?

I mean, it won't detect any `node_modules` or `yarn.lock` in the other subdirectories such as `src/docs` because it has its own `.gitignore`, right?.